pages/citations.md

```diff
@@ -87,20 +78,14 @@ We appreciate it if you display our badge in your projects that use MDAnalysis.
 
 ## <a name="IncludedAlgorithms"></a>Included algorithms ##
 
-The full list of [references for included
-algorithms]({{site.pypi.docs}}/documentation_pages/references.html#citations-for-included-algorithms-and-modules)
-is maintained and kept up-to-date as part of the documentation.
+The full list of citations for [included algorithms]({{site.pypi.docs}}/documentation_pages/references.html#citations-for-included-algorithms-and-modules) is maintained in our documentation and updated regularly.
 
 ## Acknowledgements
 
-MDAnalysis was originally inspired by the Schulten Group's
-[MDTools](http://www.ks.uiuc.edu/Development/MDTools/) for Python, and the DCD
-reading code is derived from VMD's
-[catdcd](http://www.ks.uiuc.edu/Development/MDTools/catdcd/). MDAnalysis is GPL
-licensed, except for some 3rd party code that is included under GPL-compatible
-licenses; for instance the dcd reading code is under the [UIUC Open Source
-Licence](http://www.ks.uiuc.edu/Development/MDTools/catdcd/license.html). See
-the files AUTHORS and LICENSE in the distribution for details.
+MDAnalysis was inspired by the Schulten Group’s [MDTools](http://www.ks.uiuc.edu/Development/MDTools/) for Python. The DCD reading code originates from VMD’s [catdcd](http://www.ks.uiuc.edu/Development/MDTools/catdcd/).
+
+MDAnalysis is released under the GPL license, with some third-party code under GPL-compatible licenses, such as the DCD reading code, which is licensed under the [UIUC Open Source
+Licence](http://www.ks.uiuc.edu/Development/MDTools/catdcd/license.html). See the files AUTHORS and LICENSE in the distribution for details.
```

Comment:
Could you link to [AUTHORS](https://raw.githubusercontent.com/MDAnalysis/mdanalysis/refs/heads/develop/package/AUTHORS) and [LICENSE](https://raw.githubusercontent.com/MDAnalysis/mdanalysis/refs/heads/develop/package/LICENSE) ?